cmd/ci-secret-bootstrap/main.go

```diff
@@ -794,7 +791,7 @@ func main() {
 	if err := o.validateOptions(); err != nil {
 		logrus.WithError(err).Fatal("Invalid arguments.")
 	}
-	if err := o.completeOptions(&censor); err != nil {
+	if err := o.completeOptions(&censor, o.kubernetesOptions.LoadClusterConfigs); err != nil {
```

Comment:
Why pass the function instead of calling it and passing the results ... ?

Reply:
Because of my broken brain.

Updated.